### PR TITLE
Summary getter

### PR DESCRIPTION
**Pull request by yurenpang**
get extract (summary) of each domain concept, merge all label candidates, and perform lda/hdp/lsi topic modeling on the summary texts

### CODE REVIEW

File: cartograph/topic_finder.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+    for row in article_labels_with_cluster.itertuples():
+        country = row.country
+        label_word = label_names_dic[row.label_id]
+
+        if label_word not in STOP_WORDS and not isinstance(label_word, float):
+
+            text[country].append(label_word.lower().strip())
+
+    print(text)
```
Comment (shilad) [inline cartograph/topic_finder.py]: Get rid of the print.

File: cartograph/topic_finder.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    return text
+
+
+def create_text_corpus_combined(article_labels, label_names, cluster_groups, country_labels):
```
Comment (shilad) [inline cartograph/topic_finder.py]: Get rid of this.

File: cartograph/topic_finder.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    bigram = gensim.models.Phrases(texts)
+    texts = [bigram[line] for line in texts]
+
+    dictionary = Dictionary(texts)
+    corpus = [dictionary.doc2bow(text) for text in texts]
+
+    return corpus, dictionary
+
+
+def create_text_corpus_by_csv(article_summary_csv):
```
Comment (shilad) [inline cartograph/topic_finder.py]: Comment this. A comment at the top is good.

File: cartograph/topic_finder.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    # vis = pyLDAvis.gensim.prepare(ldamodel, corpus, dictionary)
+    # pyLDAvis.save_html(vis, 'lda_vis_text.html')
+
+
+def main(directory, experiment_path):
+    corpus, dictionary = create_text_corpus_by_csv(directory + "/article_summary.csv")
+    model = run_model(corpus, dictionary, method='LDA')
+    print(model.show_topics(num_words=5))
+    # Save the model
+    model.save(experiment_path + 'topic_model')
```
Comment (shilad) [inline cartograph/topic_finder.py]: TODO: also save this as a data frame.
- article cluster probability
- cluster label probability

**Comment by yurenpang** [comment]
update the data pipeline by updating `make-culture-map.sh` and `make-food-map.sh` and `label_creator_combined.py`

**Pull request merged**